hudi-client/hudi-client-common/src/main/java/org/apache/hudi/client/BaseHoodieWriteClient.java

```diff
@@ -865,6 +874,9 @@ public HoodieCleanMetadata clean(String cleanInstantTime, boolean scheduleInline
         table.getMetaClient().reloadActiveTimeline();
       }
 
+      if (delegateToTableManagerService(config, ActionType.clean)) {
+        return null;
```

Comment:
Why is `null` returned here?

Reply:
Because the clean action has not been carried out at this time. So the cleanMetadata is null.